modules/ROOT/pages/publish-an-http-api.adoc

```diff
@@ -1,17 +1,18 @@
-= Share an HTTP API Asset
+= Publish an HTTP API Asset
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images/
+:page-aliases: to-share-an-http-api.adoc
 
 The HTTP API lets Private Exchange users specify a placeholder for an endpoint. After you create the HTTP API endpoint, you can optionally manage the endpoint with API Manager.
 
-The HTTP API asset contains a description for what the API endpoint does and how users can utilize the API for use with their programs.
+The HTTP API asset contains a description of what the API endpoint does and how users can connect the API to their programs.
```

Comment:
The HTTP API asset explains or describes? (fewer words)

Reply:
That is better. Fixed.